### PR TITLE
Import delta-connectors#559 Flink SQL documentation

### PR DESCRIPTION
**Pull request by scottsand-db**

### CODE REVIEW

File: connectors/flink/README.md [hunk 1]
```diff
@@ -41,15 +42,14 @@ Depending on the version of the connector you can use it with following Apache F
 |  0.4.x (Sink Only)  | 1.12.0 <= X <= 1.14.5 |
 |        0.5.0        | 1.13.0 <= X <= 1.13.6 |
 |        0.6.0        |      X >= 1.15.3      |
+|        0.7.0        |      X >= 1.16.1      |
```
Comment (vkorukanti) [inline connectors/flink/README.md:45]: is this still valid now?

File: connectors/flink/README.md [hunk 2]
```diff
@@ -311,27 +311,380 @@ public DataStream<RowData> createContinuousDeltaSourceUserColumns(
 }
 ```
 
+## SQL Support
+Starting from version 0.7.0 the Delta connector can be used for Flink SQL jobs.
```
Comment (vkorukanti) [inline connectors/flink/README.md:315]: same qn here.

File: connectors/flink/README.md (hunk 1 above)
Comment (scottsand-db) [reply connectors/flink/README.md]: Updated!

**Comment by scottsand-db** [comment]
Closed by https://github.com/delta-io/delta/pull/1854

**Pull request closed without merging**